.github/workflows/latest_deps.yml

```diff
@@ -0,0 +1,160 @@
+# People who are freshly `pip install`ing from the wheels we publish to PyPI will pull
+# in the latest versions of all dependencies. We used to do this in our CI runs, which
+# would detect any incompatibilities due to new package releases. Now that we use the
+# locked poetry environment, we don't have advance notice of these breakages.defaults:
```

Comment:
This would also apply if someone used the source dist on PyPI, I think?

```suggestion
# People who are freshly `pip install`ing from the PyPI will pull in the latest versions of all
# dependencies. Since most CI runs are against the locked poetry environment, run
# specifically against the latest dependencies to know if there's an upcoming breaking
# change.
```